lib/web/ui/server.go

```diff
@@ -96,3 +97,93 @@ func MakeServers(clusterName string, servers []services.Server) []Server {
 
 	return uiServers
 }
+
+// Kube describes a kube cluster.
+type Kube struct {
+	// Name is the name of the kube cluster.
+	Name string `json:"name"`
+	// Labels is a map of static and dynamic labels associated with an kube cluster.
+	Labels []Label `json:"labels"`
+}
+
+// MakeKubes creates ui kube objects and returns a list..
+func MakeKubes(clusterName string, servers []types.Server) []Kube {
+	uiKubeClusters := []Kube{}
```

Comment:
```suggestion
	uiKubeClusters := make([]Kube, 0, len(servers))
```

Reply:
i didn't make this change, b/c the len is determine by how many kube clusters there are in each server